addons/van_jsx/src/jsx-internal.d.ts

```diff
@@ -1,19 +1,19 @@
 import * as CSS from "csstype";
-import { State } from "vanjs-core";
-import { ComponentChildren, FunctionChild } from "./type";
+import { State, ChildDom } from "vanjs-core";
+import { FunctionChild } from "./type";
 
 type OriginalElement = HTMLElement;
```

Comment:
Is this line really needed. Can we simply use `HTMLElement` whenever `OriginalElement` is referred?

Reply:
@cqh963852, what's your opinion? Can we get the line removed?

Reply:
It can be removed